src/utils/source-map.js

```diff
@@ -34,6 +36,10 @@ function shouldSourceMap(): boolean {
   return prefs.clientSourceMapsEnabled;
 }
 
+function hasMappedSource(source: Source): boolean {
+  return !isOriginalId(source.id);
```

Comment:
oh sorry for the messup. it's still wip. I'll ping you again once I'm ready with this.